tests/index.js

```diff
@@ -19,25 +19,6 @@ testPdData.body = {};
 
 describe('index.js >', () => {
   describe(' createTTE tests', () => {
-    it('should return a single tte acknowledge_log_entry', () => {
```

Comment:
Instead of removing this test, can you add the case when there is no *resolve_log_entry*? This will document the expected behavior of null.